runtimes/spiritnet/src/lib.rs

```diff
@@ -342,20 +342,22 @@ parameter_types! {
 	pub const ExitQueueDelay: u32 = 2;
 	/// Minimum 16 collators selected per round, default at genesis and minimum forever after
 	pub const MinSelectedCandidates: u32 = 16;
+	/// We only allow one delegation per round.
+	pub const MaxDelegationsPerRound: u32 = 1;
 	/// Maximum 25 delegators per collator at launch, might be increased later
 	pub const MaxDelegatorsPerCollator: u32 = 25;
 	/// Maximum 1 collator per delegator at launch, will be increased later
 	pub const MaxCollatorsPerDelegator: u32 = 1;
 	/// Minimum stake required to be reserved to be a collator is 10_000
 	pub const MinCollatorStk: Balance = 10_000 * KILT;
-	/// Max stake possible to be reserved to be collator candidate is 200_000
-	pub const MaxCollatorCandidateStk: Balance = 200_000 * KILT;
 	/// Minimum stake required to be reserved to be a delegator is 1000
 	pub const MinDelegatorStk: Balance = 1000 * KILT;
 	/// Maximum number of collator candidates
 	pub const MaxCollatorCandidates: u32 = 75;
 	/// Maximum number of concurrent requests to unlock unstaked balance
 	pub const MaxUnstakeRequests: u32 = 10;
+	/// Number of blocks per year based on 12s block time
+	pub const BlocksPerYear: BlockNumber = YEARS;
```

Comment:
is this really necessary? why not use `YEARS`?

Reply:
Turns out it can be removed when using `saturated_into` 🥳

Reply:
Fixed in aa558d4